float64 speed
float64 fspeed
int64 sleep
float64 partial_open_angle
geometry_msgs/Pose pose
geometry_msgs/Pose pose1
geometry_msgs/Pose pose2